.github/workflows/tests.yml

```
@@ -171,17 +169,15 @@ jobs:
       fail-fast: false
       matrix:
         default-channel: ['defaults', 'conda-forge']
-        python-version: ['3.7', '3.8', '3.9', '3.10']
+        python-version: ['3.8', '3.9', '3.10']
         test-type: ['unit', 'integration']
         test-group: ['1', '2', '3']
         exclude:
           # exclude all but one Python version for conda-forge
-          - default-channel: 'conda-forge'
-            python-version: '3.7'
           - default-channel: 'conda-forge'
             python-version: '3.8'
           - default-channel: 'conda-forge'
-            python-version: '3.9'
+            python-version: '3.10'
```

I think this should stay 3.9:

```suggestion
            python-version: '3.9'
```